### PR TITLE
Meeting 1 updates

### PR DESCRIPTION
**Pull request by dushimsam**
## Project

Fossology/Gsoc

### Changes

-  weekly update

### CODE REVIEW

**Comment by GMishx** [comment]
@dushimsam can you please squash your commits to one? I am seeing 6 commits here with multiple merge commits.

**Comment by dushimsam** [comment]
great, i have resolved it, I think you can now merge.

**Comment by avinal** [comment]
Hey @dushimsam, please add your own sign-off.

File: docs/2022/reuse/updates/2022-06-02.md [hunk 1]
```diff
@@ -53,3 +53,53 @@ GSoC 2022 weekly update
...
+    SPDX-License-Identifier: (LGPL-2.1 AND BSD-2-CLAUSE)
+
+- **Should I change the year ranges?**
+
+    - Concerning the year ranges, we should not change the existing year ranges(and not change the end year if no substantial modification was done in the file).
+
+- Do not modify testdata files.
+### Conclusion and Further Plans
+
+- Implement discussed solution in upcoming week.
```
Comment (GMishx) [inline docs/2022/reuse/updates/2022-06-02.md:105]: Why am I seeing updates from Rohit Pandey in your branch? Any idea @dushimsam 

Please remove if irrelevant.

**Review by GMishx — CHANGES_REQUESTED** [review]

File: docs/2022/reuse/updates/2022-06-02.md (hunk 1 above)
Comment (dushimsam) [reply docs/2022/reuse/updates/2022-06-02.md]: It was caused by the new changes which were not present locally.
So I needed to pull them from this main repository first.

Comment (GMishx) [reply docs/2022/reuse/updates/2022-06-02.md]: That's why we recommend rebasing instead of pulling :-)

Try using `git pull --rebase origin main`

Comment (dushimsam) [reply docs/2022/reuse/updates/2022-06-02.md]: alright got it, sir.

**Pull request merged**